app/models/bar.rb

```diff
@@ -1,4 +1,6 @@
 class Bar < ActiveRecord::Base
+  # has_many :reviews, dependent: :destroy
```

Comment:
Ok, that's fine. When you all merge reviews with bars, just make sure to uncomment this out and write a test to make sure all reviews are destroyed when a bar is destroyed. Your team members may be doing this already!